app/views/items/browse.html.erb

```diff
@@ -1,9 +1,4 @@
-<h1><% t "browse.title", default: "Browse" %></h1>
-
-<p>
-  <%= link_to t("browse.all_link", default: "All Items in the Archive"), prefix_path(route_path),
-        class: "btn btn-lg btn-default", rel: "search" %>
-</p>
+<%= render "browse_header" %>
```

Comment:
I just realized, we probably want these to be `render_overrideable` for easier overrides in sections, right? :man_facepalming:

Reply:
oooooooo

Reply:
good catch, I always forget that :(